pom.xml

```diff
@@ -95,6 +95,18 @@
         <pirk.forkCount>1C</pirk.forkCount>
         <pirk.reuseForks>true</pirk.reuseForks>
     </properties>
```

Comment:
I'm fine with this for now in order to leverage the performance gain - however, we will need another long term solution if your PR to JNA-GMP is not rolled into a release in a timely manner.